tests/dashbio_demos/dash-alignment-chart/app.py

```diff
@@ -702,9 +699,9 @@ def customize_showid(val):
 
 
 # only declare app/server if the file is being run directly
-if 'DASH_PATH_ROUTING' in os.environ or __name__ == '__main__':
+if 'DEMO_STANDALONE' not in os.environ:
     app = run_standalone_app(layout, callbacks, header_colors, __file__)
     server = app.server
 
-if __name__ == '__main__':
+if __name__ == "__main__":
```

Comment:
If possible, could you please change these back to single quotes (since this diff isn't relevant to the PR)?